site/content/en/docs/api_sdk/cli/_index.md

```diff
@@ -230,3 +230,43 @@ by using the [label constructor](/docs/manual/basics/creating_an_annotation_task
   ```bash
   cvat-cli import task_backup.zip
   ```
+
+### Auto-annotate
+
+This command provides a command-line interface
+to the [auto-annotation API](/docs/api_sdk/sdk/auto-annotation).
+To use it, create a Python module that implements the AA function protocol.
+
+In other words, this module must define the required attributes on the module level.
+For example:
+
+```python
+import cvat_sdk.auto_annotation as cvataa
+
+spec = cvataa.DetectionFunctionSpec(...)
+
+def detect(context, image):
+    ...
+```
```

Comment:
Probably, it will be better to add a direct link to the full example from the SDK page and a specific command to use YOLO v8 example.

Reply:
> Probably, it will be better to add a direct link to the full example from the SDK page 

That example can't be used with the CLI, because it implements the function as a class rather than as a module.

> and a specific command to use YOLO v8 example.

I did list such a command. It's directly below this code block...

Reply:
> I did list such a command. It's directly below this code block...

Sorry, I missed that.